OpenSim/Examples/Plugins/BodyDragExample/osimPluginDLL.h

```diff
@@ -23,21 +23,20 @@
  * -------------------------------------------------------------------------- */
 
 // UNIX PLATFORM
-#ifndef WIN32
-
-#define OSIMPLUGIN_API
+#ifndef _WIN32
```

Comment:
I was getting an error on Appveyor without the underscore. So added it after noticing it in other files in the repo.